src/components/chipsInput/index.tsx

```diff
@@ -298,6 +298,7 @@ class ChipsInput extends Component<OwnProps, State> {
   }
 
   onFocus = () => {
+    _.invoke(this.props, 'onFocus');
```

Comment:
Please use optional chaining and pass the event (i.e. e: NativeSyntheticEvent<TextInputFocusEventData>)

Reply:
If you can do the same for 'onBlur' that will be much appreciated!

Reply:
I can't use optional chaining, because onKeyPress above is using lodash invoke.

Reply:
The type should be `NativeSyntheticEvent<TextInputFocusEventData>` otherwise it fails on TS error